src/features/game-identifier/services/game.service.js

```diff
@@ -24,21 +24,37 @@ export function getSteamAppType(httpDetailsPage) {
 export function discoverGamesFromSteamWeb(steamApps, htmlDetailsPages) {
   return htmlDetailsPages
     .map((page, i) => {
-      const releaseDate = getReleaseDate(htmlDetailsPages);
-      const developers = getDevelopers(htmlDetailsPages);
+      const { releaseDate, developers, genres, gameDescription } =
+        getAdditionalGameInfo(page);
 
       if (getSteamAppType(page) === SteamApp.validTypes.game) {
-        return Game.fromSteamApp(steamApps[i], releaseDate, developers);
+        return Game.fromSteamApp(
+          steamApps[i],
+          releaseDate,
```

Comment:
Just put the functions here, then you don't need the code in line 27-28 and the getAdditionalGameInfo function.